src/utils.ts

```diff
@@ -30,10 +30,10 @@ export const parseQueryResult = (queryString: string) => {
     parsedQuery[key] = decodeURIComponent(val);
   });
 
-  return <AuthenticationResult>{
-    ...parsedQuery,
-    expires_in: parseInt(parsedQuery.expires_in)
-  };
+  if (parsedQuery.expires_in) {
+    parsedQuery.expires_in = parseInt(parsedQuery.expires_in);
+  }
+  return <any>parsedQuery;
```

Comment:
If the input string does not contain an `expires_in` field, it should not add an `expires_in: NaN` to the returned object either. This would unnecessarily break many tests with the new `form-urlencoded` format in place.

Notice I also changed the interface of the returned object to `any`, to make the function work in different places where the result is not strictly `AuthenticationResult`.